tests/unit_tests/test_unit.py

```
@@ -649,47 +649,51 @@ def test_is_valid_url():
+            [0.5, 0.8, 0.0],
+            [0.9, 0.7, 0.6],
+        ]
     )
-    expected_mixed_scores = expected_scores.copy()
-    expected_mixed_scores[~fits_array] -= 1
-    assert np.allclose(peptide_scores_mixed_fit, expected_mixed_scores)
 
     with pytest.raises(
-        ValueError, match="`lengths` must be provided for batched input."
+        ValueError,
+        match="`lengths` must be provided for batched input.",
```

Ruff (0.15.1)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)